File: Java/Day 3/Largest.java
public class Largest{
    public static void main(String[]args){
        int[] numbers = {3,7,9,4,6,8};

        int largest = numbers[0]; 

        for(int index = 0; index < numbers.length; index++){
            if(numbers[index] > largest){
            largest = numbers[index];
            }    
    
        }

        System.out.println("The largest number is " + largest);


    }

}

File: JavaScript/Day 3/largest.js
let numbers = [5, 14, 7, 32, 9];

let largest = numbers[0];

for (let index = 1; index < numbers.length; index++) {
    if (numbers[index] > largest) {
        largest = numbers[index];
    }
}

console.log(" The largest number is:", largest);

